extensions/amp-lightbox-gallery/validator-amp-lightbox-gallery.protoascii

```diff
@@ -25,6 +25,19 @@ tags: {  # amp-lightbox-gallery
   }
   attr_lists: "common-extension-attrs"
 }
+tags: {  # <amp-lightbox-gallery>
```

Comment:
We'll need a way to also restrict this to one instance of this component per page.

Reply:
To limit a tag to one usage per page then you'll want to use `unique: true`. See [validator.proto](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L538).

Reply:
Oh awesome! Thanks a lot!